.github/workflows/build_wheels.yml

```
@@ -151,6 +176,7 @@ jobs:
           pip install bodo bodosql
       - name: Test
         run: |
+          set -exo pipefail
```

```suggestion
          set -eo pipefail
```
Had issues with this printing tokens

---

I'm actually fixing that in a separate PR. Its a different issue, so we can keep this